app/views/lists/index.html.haml

```diff
@@ -0,0 +1,15 @@
+%h2.card-title Lists:
+%h4
+  .float-center= link_to "Add list", new_user_list_path
+- if @lists.count == 0
```

Comment:
sometimes you use `.exists?`, sometimes `.count == 0`. stick to a standard.
and the standard would be `.present?`/`blank?`